docs/configuration.rst

```diff
@@ -99,6 +103,11 @@ These two configuration parameters define which redis database to use for Open e
 
 This defines extra pip packages that are going to be installed for Open edX.
 
+- ``OPENEDX_NPM_REGISTRY`` (default: ``"https://registry.npmjs.org/"``)
+
+This defines the registry from which you'll be pulling NPM packages.
```

Comment:
"... when building Docker images."

Also, we should highlight the fact that this setting can frequently be overridden by the `NPM_REGISTRY` build argument.

Reply:
And this should presumably also lose the `OPENEDX_` prefix, correct?

Reply:
By the way, the whole subsection talks about building Docker images. If we judge we need the "... when building Docker images" remark here (i.e. it isn't evident from context), we should probably also insert it in the paragraph directly above, which discusses `OPENEDX_EXTRA_PIP_REQUIREMENTS`, to which the same consideration applies. What do you think?

Reply:
> And this should presumably also lose the OPENEDX_ prefix, correct?

Yes, you're right.

> By the way, the whole subsection talks about building Docker images. If we judge we need the "... when building Docker images" remark here (i.e. it isn't evident from context), we should probably also insert it in the paragraph directly above, which discusses OPENEDX_EXTRA_PIP_REQUIREMENTS, to which the same consideration applies. What do you think?

I guess that we should group all build-related configuration settings under a single section, which would include a paragraph explaining that setting these values requires rebuilding the openedx Docker image. But this is beyond the scope of your PR. We can tackle this later.